### PR TITLE
Task #8292: Fetch and Display data for Wallet Management Screen

### PR DESCRIPTION
**Pull request by tanbtd-1747**
### Issue Link :link:
https://edu-redmine.sun-asterisk.vn/issues/8292

### Goals :soccer:
- Display all user's wallets

### CODE REVIEW

**Comment by tanbtd-1747** [comment]
ready

**Review by longvt96 — APPROVED** [review]

**Review by minhduc2511 — APPROVED** [review]

File: Money*/Money*/Scenes/WalletManagement/WalletManagementViewController.swift [hunk 1]
```diff
@@ -7,30 +7,66 @@
...
+                    self?.presentErrorAlert(title: Constant.titleError, message: Constant.messageSnapshotError)
+                    return
+                }
+                
+                let models = snapshot.documents.map({ (document) -> Wallet in
+                    if let model = Wallet(snapshot: document) {
+                        return model
+                    } else {
+                        self?.presentErrorAlert(title: Constant.titleError, message: Constant.messageDataError)
+                        return Wallet(name: "", type: .other, balance: 0)
```
Comment (tienpx3010) [inline Money*/Money*/Scenes/WalletManagement/WalletManagementViewController.swift]: Sai logic

File: Money*/Money*/Scenes/WalletManagement/WalletManagementViewController.swift [hunk 2]
```diff
@@ -7,30 +7,66 @@
...
+        super.viewDidAppear(animated)
+        fetchData()
+    }
+    
     private func configureSubviews() {
         title = Constant.sceneTitleWalletManagement
     }
+    
+    private func fetchData() {
+        Firestore
```
Comment (tienpx3010) [inline Money*/Money*/Scenes/WalletManagement/WalletManagementViewController.swift]:
```suggestion
        Firestore.firestore()
                 .collection(user.email)
```

**Comment by tanbtd-1747** [comment]
ready

**Review by minhduc2511 — APPROVED** [review]

**Review by tienpx3010 — APPROVED** [review]

**Pull request merged**